download_sdl2_hg.cmake

```diff
@@ -27,7 +27,8 @@ endif()
 ExternalProject_Add(
     SDL2HG
     PREFIX ${CMAKE_BINARY_DIR}/external/SDL2
-    GIT_REPOSITORY https://github.com/spurious/SDL-mirror.git
+    URL https://hg.libsdl.org/SDL/archive/default.tar.bz2
+    # GIT_REPOSITORY https://github.com/spurious/SDL-mirror.git
```

Comment:
LOL :rofl: 
Are you wanted to put GIT repo in a top? I have asked you to keep old URL as comment, but you are did opposite: you have kept old URL but temporary GIT as comment! :rofl: :fox_face: